src/dataTypeCheck.js

```diff
@@ -0,0 +1,15 @@
+export default function dataTypeCheck(data,expectedType) {
+  if(expectedType === 'array' && Array.isArray(data)) {
+    return true
+  }
+  if(expectedType === 'object' && typeof data === 'object' && !Array.isArray(data)) {
+    return true
+  }
+  if(expectedType === 'string' && typeof data === 'string') {
+    return true
+  }
+  if(expectedType === 'number' && !isNaN(data)) {
+    return true
+  }
+  return false
```

Comment:
I would try to refactor this function and reduce the amount of "if" statements. Define concisely what value you are looking for and if that value type does not match then return "wrong value type". 

You could possibly get rid of it all together by just defining the specific datatype you are looking for and all other responses that do not match with return "wrong datatype" 

``` const dataTypeCheck = //acceptable array```